components/partials/JobOffersWidgetItem.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <div class="job-offers-widget-item widget-item">
+    <span :class="['job-offers-widget-item__technology-icon', devIconClass]" :aria-hidden="true"></span>
+    <div class="job-offers-widget-item__left-side">
+      <span class="job-offers-widget-item__offer-title job-offers-widget-item__left-side-element">
+         {{ upper(title) }}
+      </span>
+      <a target="_blank" :href="employerWebpage" class="job-offers-widget-item__employer-link job-offers-widget-item__left-side-element">
```

Comment:
Maybe use `v-if` and display link only when specified. I think employer name should be required to add a job offer, but employer website not.

Reply:
done